### PR TITLE
[WIP] fix: Enable PDF plugin

### LINKED ISSUE #14341: gn: add missing enable_pdf_viewer build flag
It is present in the features.gypi but not in the BUILD.gn.

### PR DESCRIPTION
**Pull request by sethlu**
Resolves https://github.com/electron/electron/issues/12337
Resolves #14341

##### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

Notes: re-enable pdf viewer

### CODE REVIEW

File: features.gypi [hunk 1]
```diff
@@ -4,7 +4,7 @@
     'variables': {
       'enable_desktop_capturer%': 1,
       'enable_osr%': 0,  # FIXME(alexeykuzmin)
-      'enable_pdf_viewer%': 0,  # FIXME(deepak1556)
+      'enable_pdf_viewer%': 1,  # FIXME(deepak1556)
```
Comment (alexeykuzmin) [inline features.gypi]: The FIXME comment should probably be removed =)

**Comment by jkleinsc** [comment]
GN builds will need https://github.com/electron/pdf-viewer/pull/14 (WIP).

**Comment by sethlu** [comment]
As we're moving into the GN build, there will be a future PR to bring back the PDF viewer with GN.

**Comment by MiXT4PE** [comment]
@sethlu What does GN mean?

**Pull request closed without merging**